graphql-client/src/index.js

```diff
@@ -100,8 +100,12 @@ export function createApolloClient ({
     }
 
     if (!disableHttp) {
-      if (persisting) {
-        link = createPersistedQueryLink().concat(link)
+      let persistingOpts = {}
+      if (typeof persisting === 'object' && persisting != null) {
+        persistingOpts = persisting
+      }
+      if (persistingOpts.enabled) {
```

Comment:
If the option `persisting` has been set to `true` the persisted query link will not be enabled. The documentation implies otherwise and this also breaks setups from previous versions.

Reply:
Oh.. I'll create a PR in a few mins

Reply:
Done https://github.com/Akryum/vue-cli-plugin-apollo/pull/241

Reply:
@Akryum please take a look on this